src/services/treeBuilder.ts

```diff
@@ -91,6 +96,8 @@ function createHierarchy(src: TreeNode): File[] {
         );
       } else {
         const [paths, relativePath] = (childrenOrFileData as unknown) as [DiffPaths, string];
+        const hasParsableContent = shouldParseFile(paths[0]) || 
```

Comment:
Let's have a upper check if this config even exists to avoid checking each file when it's not relevant at all